### PR TITLE
fix NuGet/Home#813 by honoring the NUGET_PACKAGES environment variable

### PR DESCRIPTION
**Pull request by analogrelay**
Looks like VS is fine, this just adds command line support to the prototype command line

/cc @emgarten @yishaigalatzer @pranavkm

### CODE REVIEW

File: src/NuGet.CommandLine/Program.cs [hunk 1]
```diff
@@ -97,19 +97,30 @@ public int Main(string[] args)
...
+                            {
+                                request.PackagesDirectory = packagesDirectory.Value();
+                            }
+                            else
+                            {
+                                request.PackagesDirectory = SettingsUtility.GetGlobalPackagesFolder(settings);
+                            }
+
+                            // Resolve the packages directory
+                            _log.LogVerbose($"Using packages directory: {request.PackagesDirectory}");
```
Comment (pranavkm) [inline src/NuGet.CommandLine/Program.cs:144]: Localized?

File: src/NuGet.Commands/RestoreRequest.cs [hunk 2]
```diff
@@ -14,17 +13,22 @@ public class RestoreRequest
     {
         public static readonly int DefaultDegreeOfConcurrency = 8;
 
+        public RestoreRequest(PackageSpec project, IEnumerable<PackageSource> sources)
+            : this(project, sources, null)
```
Comment (pranavkm) [inline src/NuGet.Commands/RestoreRequest.cs]: named param.

**Comment by pranavkm** [comment]
:shipit:

File: src/NuGet.CommandLine/Program.cs (hunk 1 above)
Comment (yishaigalatzer) [inline src/NuGet.CommandLine/Program.cs:144]: :+1:

Comment (analogrelay) [inline src/NuGet.CommandLine/Program.cs:144]: I'll localize YOU!

Comment (analogrelay) [inline src/NuGet.CommandLine/Program.cs:144]: This is in the command line, which isn't localized right now.

**Comment by emgarten** [comment]
:shipit:

**Pull request closed without merging**